src/index.ts

```diff
@@ -1,26 +1,42 @@
-import { ReplaySubject } from 'rxjs'
+import { Observable, ReplaySubject } from 'rxjs'
 
-type SubjectByProp = Map<string, ReplaySubject<any>>
+interface Cache { observable: Observable<any>; subject: ReplaySubject<any> }
```

Comment:
`Cache` isn't a great name, it would be a `CacheEntry` if it was stored in a cache, but semantically, what it is stored in is not a cache. Maybe `SubjectAndObservable` or something?

Reply:
Alright, I head SubjectAndObservable before and changed because it was too long. :-) But I'm happy to change it back. I also changed the other names accordingly.